### PR TITLE
Document the interpreter metadata on content in the Partner v3 API

### PR DESCRIPTION
**Pull request by kivra-fabber**
Update docs to describe the new interpreter meta properties we added July 31.

See issues K4B-2015, K4B-2097.

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -3063,6 +3067,27 @@ paths:
...
+        Interpreted data is never used in place of metadata provided by the sender or Kivra, it can only be added.
+
+        Any property of the response that was added by the interpreter is clearly identified as such.
+        If an interpreted property is an object, it will have an `interpreted` property which is `true`.
+        If an interpreted property is a primitive value, its key path will be present in the `interpreted_fields` property.
+        See the response schema below for details.
+
+        ### Content file data
+
+        A given content can contain multiple files ("parts"), to download one, pass the `part` query parameter.
```
Comment (kivra-ponern) [inline swagger.yml:3086]:
```suggestion
        A given content can contain multiple files ("parts"). Fetching of such parts is done by passing the `part` query parameter.
```
Not sure "download" is the word to use.

File: swagger.yml [hunk 2]
```diff
@@ -3063,6 +3067,27 @@ paths:
...
       summary: Get specific content
+      description: |
+        ### Content metadata
+
+        Get metadata about a specific content in JSON format.
+
+        If the interpreter service is enabled for a recipient, it can add structured metadata to content which has been received.
+        The `interpreted` label indicates if the interpreter has run for a given content.
+        Interpreted data is never used in place of metadata provided by the sender or Kivra, it can only be added.
+
```
Comment (kivra-ponern) [inline swagger.yml:3078]: Consider joining these two paragraphs.
```suggestion
```

File: swagger.yml [hunk 3]
```diff
@@ -5113,32 +5165,35 @@ components:
...
+          description: The name of the sender
+          example: Lenas Konditori AB
           type: string
+          nullable: true
         sender_vat_numbers:
+          description: VAT numbers representing the sender
           type: array
           items:
             $ref: '#/components/schemas/VatNumber'
+          nullable: true
```
Comment (kivra-ponern) [inline swagger.yml:5193]: And just to verify it's `null` and not empty array `[]`?

File: swagger.yml [hunk 4]
```diff
@@ -5113,32 +5165,35 @@ components:
...
         subject:
+          description: Subject of the content
+          example: "Faktura: tusen rulltårtor"
```
Comment (kivra-ponern) [inline swagger.yml:5196]: Would be nicer to keep the examples in English.
```suggestion
          example: "Invoice: a thousand Swiss rolls"
```
:laughing:

File: swagger.yml [hunk 5]
```diff
@@ -4940,61 +4981,62 @@ components:
...
           example: SEK
-          oneOf:
-            - type: string
-            - type: nil
+          type: string
+          nullable: true
         description:
-          description: The description of this payment, if available
+          description: The description of this payment
           example: Takplåt
```
Comment (kivra-ponern) [inline swagger.yml:4999]: English please.

File: swagger.yml [hunk 6]
```diff
@@ -3022,9 +3022,13 @@ paths:
...
       summary: List company inbox content
       operationId: GET_company-companyKey-content
-      description: >
+      description: |
         List the content this company has received.
         This may be filtered by certain `labels` and specific fields may be selected using `field`.
+
+        Since the list can be long, not all details about each content are in the response.
+        More detail is available via the
+        [Get specific content](#tag/Partner-API-v3/operation/Get%20content) endpoint.
```
Comment (kivra-ponern) [inline swagger.yml:3031]:
```suggestion
        Note that more content metadata is available via the
        [Get specific content](#tag/Partner-API-v3/operation/Get%20content) endpoint.
```
Disclaimer reads weirdly imo.

**Review by kivra-ponern — CHANGES_REQUESTED** [review]
Nitpicks :hamster:

File: swagger.yml (hunk 3 above)
Comment (kivra-fabber) [reply swagger.yml:5193]: Correct, if not available from the sender, or the interpreter, it will be null.

**Review by kivra-ponern — APPROVED** [review]

**Pull request merged**